src/routes/index.ts

```diff
@@ -20,6 +20,15 @@ export class Routes {
 				limit: Number(limit) || 0,
 			};
 
+			// add ssl filter to query isHttpsEnabled nodes.
+			if (ssl) {
+				const isSSL = ssl.toString().toLocaleLowerCase() === 'true';
```

Comment:
what if the ssl is undefined?

Reply:
the list will return http and https together